tests/Issues/AddNodeAfterNodeStmt/Source/AddNextStmtRector.php

```diff
@@ -34,17 +28,9 @@ public function getNodeTypes(): array
 
     public function refactor(Node $node)
     {
-        if (isset($this->justAdded[$this->file->getFilePath()])) {
-            return null;
-        }
-
-        $this->nodesToAddCollector->addNodeAfterNode(
+        return [
```

Comment:
This test is on purpose for add node after node with inline html usages, so need to be reverted.